### ISSUE TITLE
Inputs broken with WP 4.4

### ISSUE THREAD

**Issue by BeatLaG** [elliotcondon/acf#477]
Hello,

I´m checking a WP theme I use to use in some projects and today I have installed it within a WP 4.4 clean installation and the inputs seem to be broken. Please, check the attached file.

Reviewing this issue with another 3.x installation, the fields look well but upgrading the WP version these elements appear in that way. Any idea about it?

Thanks!

PS. They should look 100% width

![acf-wp-44-issue](https://cloud.githubusercontent.com/assets/1290405/11481280/ce0ab2f4-979c-11e5-92d5-318c3ae36261.png)

**Comment by elliotcondon** [comment]
Hi @BeatLaG 

I've just tested this and can't replicate the issue.
Please check the inspector to view the CSS applied to the repeater table, perhaps you have another plugin / theme style which is modifying the width?

**Comment by BeatLaG** [comment]
I´ll follow debugging this issue then. It is pretty weird because the feature was working before updating to WP4.4 RC. I´ll contact the theme developers as well in order to have another opinion.

Thank you very much, Elliot

**Comment by elliotcondon** [comment]
All good, let me know what you find in your console log CSS

**Comment by BeatLaG** [comment]
Hi Eliot,

nothing at the console, looks like some kind of problem with the current session opened in Chrome. For some weird reason, with cache cleaned, the repeater were looking like that. I have reinstalled Chrome in my main machine and everything looks great again.

I noticed this with my laptop turning this weird issue in something really crazy to troubleshoot because no CSS was affecting to this.

Well, no worries because everything is working perfectly now. It was my bad for not having this into account.

Thanks anyway!
